packages/addons/service/openvfd-service/package.mk

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="openvfd-service"
+PKG_VERSION="99b2ab997623ccc7ee6914b1b832514e36a65eda"
+PKG_SHA256="23213823d1a602217c09998ec49409ca17113c2a318541cf53e9333d8e6a169b"
+PKG_REV="100"
+PKG_ARCH="arm aarch64"
+PKG_LICENSE="GPL2"
+PKG_SITE="https://github.com/arthur-liberman/service.openvfd"
+PKG_URL="https://github.com/arthur-liberman/service.openvfd/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="service"
+PKG_SHORTDESC="OpenVFD: Service for VFD displays"
+PKG_LONGDESC="OpenVFD: Service for controlling Ethernet and WiFi status, Time, Date and Playback indicators via the Linux OpenVFD driver"
+PKG_TOOLCHAIN="manual"
+
+PKG_IS_ADDON="yes"
+PKG_ADDON_NAME="OpenVFD"
+PKG_ADDON_TYPE="xbmc.service"
+PKG_ADDON_PROJECTS="Allwinner Amlogic Rockchip"
+
+post_unpack() {
+  rm $PKG_BUILD/addon.xml
```

Comment:
We allow Kodi binary add-ons to follow their own versioning, but all other add-ons in the repo use addon.xml/changelog.txt that we build since the add-on is under our version control. The way we normally reflect the upstream version is pulling the sources by version and adding $PKG_VERSION to the $PKG_SHORTDESC. In this case I'm using a long githash for version as the latest 'release' on GitHub is several months behind latest commits in the repo - I've sent you a note on that and some license points via forum PM. We also use icons that follow a consistent house style. If possible I adapt the original icon or project/app logo, but that would be hard with the current upstream icon.png so I've reused/adapted the art from the original fd628 add-on.

Reply:
That would be a total no-go for me.
1. You're de-branding/rebranding the addon.
1. You're removing the original author.
1. You're getting rid of the change log and any history of changes.